standalone/pruntime/enclave/src/contracts/btc_lottery.rs

```diff
@@ -0,0 +1,441 @@
+                if (self.token_set.contains_key(&round_id)
+                    && self.lottery_set.contains_key(&round_id))
+                {
+                    let token_number = self
+                        .token_set
+                        .get(&round_id)
+                        .expect("round_id is invalid in token_set")
+                        .len();
+                    let winner_count = self
+                        .lottery_set
+                        .get(&round_id)
+                        .expect("round_id is invalid in lottery_set")
+                        .len();
+                    Response::GetRoundInfo {
+                        token_number: token_number as u32,
+                        winner_count: winner_count as u32,
+                    }
+                } else {
+                    Response::GetRoundInfo {
+                        token_number: 0,
+                        winner_count: 0,
+                    }
+                }
+            }
+            Request::QueryUtxo { round_id } => {
+                if (self.utxo.contains_key(&round_id)) {
+                    Response::QueryUtxo {
+                        utxo: self
+                            .utxo
+                            .get(&round_id)
+                            .expect("round_id is invalid in utxo")
+                            .clone(),
+                    }
+                } else {
+                    Response::QueryUtxo {
+                        utxo: BTreeMap::<Address, (Txid, u32, u64)>::new(),
+                    }
+                }
+            }
+            Request::PendingLotteryEgress { sequence } => {
+                println!("PendingLotteryEgress");
+                let transfer_queue: Vec<&SendLotteryData> = self
+                    .queue
+                    .iter()
+                    .filter(|x| x.data.sequence > sequence)
+                    .collect::<_>();
+
+                Response::PendingLotteryEgress {
+                    lottery_queue_b64: base64::encode(&transfer_queue.encode()),
+                }
+            }
+        }
+    }
+
+    fn handle_event(&mut self, ce: chain::Event) {
+        if let chain::Event::pallet_bridge_transfer(pe) = ce {
+            if let chain::pallet_bridge_transfer::Event::LotteryNewRound(
+                round_id,
+                total_count,
+                winner_count,
+            ) = pe
+            {
+                let raw_seed = self.secret.as_ref().unwrap().to_raw_vec();
+                let token_id: U256 = U256::from(round_id) << 128;
+                let mut round_token = match self.token_set.get(&round_id) {
+                    Some(_) => self
+                        .token_set
+                        .get(&round_id)
+                        .expect("invalid round_id in token_set")
+                        .clone(),
+                    None => Vec::new(),
+                };
+                for index_id in 0..total_count {
```

Comment:
Let use [1, total_count] instead of [0, total_count)

Reply:
Already fixed